### DISCUSSION TITLE
GitHub like preview.

### DISCUSSION THREAD

**Discussion by DarshitSolanki04** [KDE/ghostwriter#812] [General]
Is there any way to generate *GitHub like preview* in Ghostwriter? It would be nice if there is some way to generate GitHub like output in *HTML* and *pdfs*.

@wereturtle

**Comment by ferrlen** [comment]
At least in the latest version there's an "Export" option under "File" that exports to HTML. There's also a "Live HTML Preview" panel that can be toggled in the lower-right corner of the screen (it's the one with a "</>" symbol)
